xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -327,20 +327,6 @@ bool CSystemGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
       return true;
     }
 
-    case SYSTEM_LOCALE_TIMEZONECOUNTRY:
```

Comment:
SYSTEM_LOCALE_TIMEZONECOUNTRY needs  also to be removed or am I missing something from the diff?

Reply:
Same for SYSTEM_LOCALE_TIMEZONE?

Reply:
Also, since both info's are removed it needs documentation and skin revision tags

Reply:
Yes, you're quite right, we should take care of this, too.

So it seems this was added only pretty recently in #24028. I'm not sure what exactly the use case is for the timezone that, AFAICT, was only ever available on Linux, but well it is in for Nexus now. It seems a bit cruel to rip it out again after just one version, but I guess it is what it is :/

I haven't dealt with skin revision tags before, would it help to just return empty strings so that old skins don't break? or is that the default anyway when an infolabel vanishes?